### DISCUSSION TITLE
support for panels

### DISCUSSION THREAD

**Discussion by emard** [kitspace/kitspace#362] [General]
Is there some recommendation about how
to integrate PCB repository that uses kitspace with ordering
of PCB panels and stencils for multiple PCB production, 
in the case when repository author creates his own panel design
instead of letting manufacturer do it

For example I have ulx3s project and I have used "kikit"
panelizer to arrange 8 PCBs on each panel so I''d like to
click order from JLCPCB and panel to be submitted
with some default options (quality)
so I can quickly get price estimate without need to
click every option again on JLC website

**Comment by kasbah** [comment]
Hi Emard, indeed we need to figure something out here. I don't have any recommendations for the moment. The only thing you _can_ do right now is make another project with the panel. 

I think it's a good idea to try and integrate the KiKit panelization to allow users to order custom panels. I will reach out to @yaqwsx to see if we can work together on this.

**Comment by emard** [comment]
That would be great!

when parts reach out of board edges, it needs some thinking
and care about panelization, it's best that project author
designs the panel instead of manufacturer

kikit works really great and it's close to 1.0 release

**Comment by takeoff-projects-final-year** [comment]
Really good information about <a href="https://takeoffprojects.com/simple-electronics-projects">[Simple Electronics Projects](https://takeoffprojects.com/simple-electronics-projects)</a>.
